ansible_collections/arista/avd/roles/eos_cli_config_gen/tasks/main.yml

```diff
@@ -27,38 +27,36 @@
 
 - name: Include device intended structure configuration variables
   tags: [build, provision]
-  arista.avd.include_vars:
+  include_vars:
     file: "{{ filename }}"
   delegate_to: localhost
   # errors='ignore' is needed for compatibility with ansible-core < 2.12
   when: structured_config is not defined and lookup('first_found', filename, skip=True, errors='ignore')
   vars:
     filename: "{{ structured_dir }}/{{ inventory_hostname }}.{{ avd_structured_config_file_format }}"
 
-- name: Validate & update input vars according to AVD eos_cli_config_gen schema
-  # This task should not be skipped when using AVD v3.x input data,
-  # since it is responsible for upgrading input data to v4.0 models.
-  tags: [always, validate]
-  arista.avd.validate:
-    schema: "{{ lookup('ansible.builtin.file', role_schema_path) | from_yaml }}"
-    conversion_mode: "{{ avd_validate_conversion_mode }}"
-    validation_mode: "{{ avd_validate_validation_mode }}"
-  delegate_to: localhost
-
 - name: Generate eos intended configuration
   tags: [build, provision]
```

Comment:
so now that the validate "which should not be skipped using AVD v3.x" is not having the `always` tags - could there be situation where people are going to skip it?

Reply:
Well we no longer care, since it is part of every module that needs it. We will still respect the `avd_validate_conversion_mode` so if people want to break it, they can :)